pkg/extensions/search/digest_test.go

```diff
@@ -1,28 +1,22 @@
-//go:build search
```

Comment:
You probably need to keep the build labels, as there tests would probably not work if the rest of the search code is not included in the build.